### PR TITLE
docs: python-tox no longer exists in Debian Buster

### PR DESCRIPTION
**Pull request by eyalch**

### CODE REVIEW

**Comment by encukou** [comment]
I assume it'll be the same in Debian Bullseye and newer ones. Right?
I think it would make sense to put `tox` in the instructions and have a note for *older* Debian versions.

File: Doc/installing.rst [hunk 1]
```diff
@@ -148,6 +148,8 @@ Packages for building and testing::
    # apt-get install build-essential python3-dev python2.7-dev \
        libldap2-dev libsasl2-dev slapd ldap-utils python-tox \
```
Comment (tiran) [inline Doc/installing.rst:149]:
```suggestion
       libldap2-dev libsasl2-dev slapd ldap-utils tox \
```

File: Doc/installing.rst [hunk 2]
```diff
@@ -148,6 +148,8 @@ Packages for building and testing::
...
        lcov valgrind
+       
+*Note:* Replace ``python-tox`` with ``tox`` in Debian Buster.
```
Comment (tiran) [inline Doc/installing.rst:152]:
```suggestion
.. note::
   
   On older releases ``tox`` was called ``python-tox``.
```

**Comment by tiran** [comment]
Travis CI is blocked

**Pull request merged**